internal/log/log.go

```diff
@@ -36,12 +41,29 @@ var (
 	once   sync.Once
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
where's the CRIT level in Logger interface you've defined?
https://github.com/vdaas/vald/pull/186/files#diff-3e8fb82147cb0e3ccfa83d2b067acc5bR39

Reply:
Although it doesn't exist at present, it will match to the interface of glg. However, I will comment out once because there is no plan to use it now.